app/_webparts/webparts/model-listing/style.scss

```diff
@@ -0,0 +1,76 @@
+.humanise-model-listing {
+
+	.heading-type-3 {
+		font-size: 1.41176470588em;
+		margin-bottom: 10px;
+		
+		@media (min-width: $screen-md) {
+			margin-bottom: 1em;
+		}
+		
+		h2 {
+			margin-bottom: 10px;
+			
+			@media (min-width: $screen-md) {
+				margin-bottom: 1em;
+			}		
+		}
+	}
+
+	.humanise-model-listing__list-item {
+		border-bottom: 1px solid $grey-light;
+		display: block;
+		overflow: hidden;
+		padding-bottom: 0;
+		margin-bottom: 1em;
+		
+		@media (min-width: $screen-md) {
+			padding-bottom: 1.5em;
+			margin-bottom: 1.5em;		
+		}
+	}
+	
+	.humanise-model-listing__info {
+		display: block;
+		overflow: hidden;
+	}
+
+	.humanise-model-listing__details {
+		
+		@media (min-width: $screen-md) {
+			float: left;
+			width: 50%;		
+		}
+	}
+	
+	.humanise-model-listing__image {
+		margin-bottom: 1em;
+		
+		@media (min-width: $screen-md) {
+			float: right;
+			width: 50%;				
+		}
+	}
+	
+	.humanise-model-listing__features {
+		margin: 0 0 2em 0;
+		padding: 0;
+		display: block;
+		
+		li {
+			position: relative;
+			padding-left: 20px;
+			font-family: $skoda-next-light;
+			&:before {
```

Comment:
nice - but can we add that this code to this ticket for the molecule - https://rippleffect.atlassian.net/browse/SKMW-1795

I was thnking I needed Jim to do svgs but your way makes much more sense!